service/server.py

```diff
@@ -239,6 +250,10 @@ def _show_full_title_data():
     return app.config.get('SHOW_FULL_TITLE_DATA')
 
 
+def _show_full_title_pdf():
```

Comment:
How about `_should_show_full_title_pdf()`? _show... sounds like it actually does it.